bp-app-api/src/main/java/com/beautify_project/bp_app_api/service/ShopService.java

```diff
@@ -4,51 +4,79 @@
 import com.beautify_project.bp_app_api.dto.shop.ImageFiles;
 import com.beautify_project.bp_app_api.dto.shop.ShopFindListRequestParameters;
 import com.beautify_project.bp_app_api.dto.shop.ShopFindResult;
+import com.beautify_project.bp_app_api.dto.shop.ShopListFindResult;
 import com.beautify_project.bp_app_api.dto.shop.ShopRegistrationRequest;
+import com.beautify_project.bp_app_api.dto.shop.ShopRegistrationRequest.IdName;
 import com.beautify_project.bp_app_api.dto.shop.ShopRegistrationResult;
+import com.beautify_project.bp_app_api.entity.Facility;
+import com.beautify_project.bp_app_api.entity.Operation;
+import com.beautify_project.bp_app_api.entity.Shop;
+import com.beautify_project.bp_app_api.repository.CategoryRepository;
+import com.beautify_project.bp_app_api.repository.FacilityRepository;
+import com.beautify_project.bp_app_api.repository.OperationRepository;
+import com.beautify_project.bp_app_api.repository.ShopRepository;
 import java.util.Arrays;
+import java.util.Base64;
 import java.util.List;
+import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
 import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
 
 @Service
 @Slf4j
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
 public class ShopService {
 
+    private final ShopRepository shopRepository;
+    private final StorageService storageService;
+    private final FacilityRepository facilityRepository;
+    private final OperationRepository operationRepository;
+    private final CategoryRepository categoryRepository;
```

Comment:
```suggestion
    private final StorageService storageService;
    private final FacilityRepository facilityRepository;
    private final OperationRepository operationRepository;
    private final CategoryRepository categoryRepository;
```

ShopService에서 다른 리포지토리들을 보는 것에 대해서

Reply:
서비스를 만들어서 분리하는 것으로

Reply:
해당 내용은 8주차 화상 멘토링에서 말씀드렸다시피 아직 구현하지 않아 위와 같이 작성했었습니다.
추후 서비스 프로덕션/테스트 코드 리팩토링 PR에 반영하겠습니다!